### PR TITLE
Change Microsoft.Framework.*  to Microsoft.Extensions.*

### PR DESCRIPTION
**Pull request by ORuban**
Change Microsoft.Framework.\*  to Microsoft.Extensions.\* accordingly to the following annoubcement:
https://github.com/aspnet/Announcements/issues/77

### CODE REVIEW

File: samples/latest/HelloWeb/project.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "Microsoft.AspNet.Server.IIS": "1.0.0-*",
```
Comment (pranavkm) [inline samples/latest/HelloWeb/project.json]: Slightly unrelated, but could you also remove this dependency? We no longer produce this package.

Comment (ORuban) [inline samples/latest/HelloWeb/project.json]: Done, please check.

**Comment by Kagamine** [comment]
Can you add `app.UseIISPlatformHandler();` into sample?

**Comment by ORuban** [comment]
@Kagamine I would like to suggest creating another PR for that...  Seems like another scope of changes + I am not familiar with UseIISPlatformHandler

**Comment by Eilon** [comment]
@glennc

**Pull request closed without merging**